usr/share/rear/prep/BACULA/default/500_check_BACULA_bconsole_results.sh

```diff
@@ -29,11 +29,11 @@ fi
 # and that the director can connect to the file daemon on this system.
 # "Connecting to Director 'director_name-fd:9101'"
 # "Connecting to Client 'bacula_client_name-fd at FQDN:9102"
-BACULA_CLIENT=`grep $(hostname -s) /etc/bacula/bacula-fd.conf | grep "\-fd" | awk '{print $3}' | sed -e "s/-fd//g"`
+BACULA_CLIENT=$(grep $(hostname -s) $BACULA_CONF_DIR/bacula-fd.conf | grep "\-fd" | awk '{print $3}' | sed -e "s/-fd//g")
 [ "${BACULA_CLIENT}" ]
-StopIfError "Client $(hostname -s) not defined in /etc/bacula/bacula-fd.conf"
+StopIfError "Client $(hostname -s) not defined in $BACULA_CONF_DIR/bacula-fd.conf"
```

Comment:
As far as I see that
```
StopIfError "Client $(hostname -s) not defined ..."
```
errors out only when `hostname -s` results non-zero exit code
but not when `[ "${BACULA_CLIENT}" ]` results non-zero exit code, cf.
https://github.com/rear/rear/blob/master/usr/share/rear/lib/_input-output-functions.sh#L732